site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -326,6 +326,40 @@ The password minimum age in seconds. When enabled FusionAuth will not allow a pa
 [field]#{base_field_name}.minimumPasswordAge.enabled# [type]#[Boolean]# [since]#Available since 1.8.0#::
 Indicates that the minimum password age is enabled and being enforced.
 
+[field]#{base_field_name}.multiFactorConfiguration.authenticator.algorithm# [type]#[String]# [since]#Available since 1.26.0#::
+The algorithm used by the TOTP authenticator. The possible values are:
++
+* `HmacSHA1`
+* `HmacSHA256`
+* `HmacSHA512`
+
+[field]#{base_field_name}.multiFactorConfiguration.authenticator.codeLength# [type]#[Integer]# [since]#Available since 1.26.0#::
+The length of code generated by the TOTP.
+
+[field]#{base_field_name}.multiFactorConfiguration.authenticator.enabled# [type]#[Boolean]# [since]#Available since 1.26.0#::
+When enabled, users may utilize an authenticator application to complete a multi-factor authentication request. This method uses TOTP (Time-Based One-Time Password) as defined in link:https://tools.ietf.org/html/rfc6238[RFC 6238] and often uses an native mobile app such as Google Authenticator.
++
+Currently the only valid value for algorithm is `HmacSHA1`, for `codeLength` is `6`, for `timeStep` is `30`. Other values will be ignored.
```

Comment:
This kind of sounds like this belongs on the request instead of the response?

Reply:
Agreed. Since I made documentation in the response that those fields are read-only and specified the values, I'm just going to omit this all together from the request.